src/index.ts

```diff
@@ -0,0 +1,8 @@
+function helloworld (msg = 'Hello World!'): string {
+  console.log(msg)
+  return msg
+}
+
+helloworld()
+
+export default helloworld
```

Comment:
Why you exported this function ? No one is using it.

Reply:
I needed to export it so that I could use it on tests.